libs/server/Resp/Parser/SessionParseState.cs

```diff
@@ -46,8 +44,7 @@ public unsafe struct SessionParseState
         /// <summary>
         /// Initialize the parse state at the start of a session
```

Comment:
Change `public readonly Span<ArgSlice> Parameters` to `public ReadOnlySpan<ArgSlice> Parameters`